test/parallel/test-buffer-readintbe.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readIntBE()
+
+const buf = Buffer.from([42, 84, 167, 127]);
+const result = buf.readIntBE(1);
+
+assert.strictEqual(result, 42);
+
+assert.throws(
+  () => {
+    buf.readIntBE(5);
+  },
+  /^RangeError: Index out of range$/
+);
+
+assert.doesNotThrow(
+  () => {
+    buf.readIntBE(5, 0, true);
```

Comment:
Second argument must be greater than `0`, so please change it.  (I'd probably use `1`. You can use any integer up to and including `6`.)